File: app/src/main/java/com/asolis/kotlinmvvmbase/ui/base/BaseFragment.java
package com.asolis.kotlinmvvmbase.ui.base;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.asolis.kotlinmvvmbase.ui.ActivityListener;


public abstract class BaseFragment extends Fragment implements ActivityListener {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
}

File: app/src/main/java/com/asolis/kotlinmvvmbase/ui/base/BaseFragment.kt
package com.asolis.kotlinmvvmbase.ui.base

import android.os.Bundle
import androidx.fragment.app.Fragment
import com.asolis.kotlinmvvmbase.ui.ActivityListener

abstract class BaseFragment : Fragment(), ActivityListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }
}